src/i18n/en.json

```diff
@@ -123,6 +123,10 @@
 		"BUTTON_ARIA_LEFT": "Go to the previous tab",
 		"BUTTON_ARIA_RIGHT": "Go to the next tab"
 	},
+	"TOGGLE": {
+		"LEFTTEXT": "Off",
```

Comment:
```suggestion
		"OFF": "Off",
```

Reply:
If design changes so off state is on the right, things shouldn't break. Technically toggle has an off and an on state so let's use those names.